miasm2/arch/mips32/arch.py

```diff
@@ -98,7 +98,7 @@ def dstflow2label(self, symbol_pool):
             expr = self.args[0].arg
             addr = (self.offset & (0xFFFFFFFF ^ ((1<< 28)-1))) + expr
             label = symbol_pool.getby_offset_create(addr)
-            self.args[0] = ExprLoc(label.loc_key, expr.size)
+            self.args[0] = ExprLoc(label, expr.size)
```

Comment:
Nice catch!
Can you rename as well the "label" to "loc_key" to reflect the object type in the patch ?
Thank you!

Reply:
Done.

This was discovered thanks to a r2m2 unit test =)